app/config/parameters_test.yml.dist

```diff
@@ -8,3 +8,6 @@ parameters:
     locale:            en
     secret:            ThisTokenIsNotSoSecretChangeIt
     installer_data:    PimInstallerBundle:minimal
+    pim_es_index_name: product_index
```

Comment:
maybe "behat_product_index" or something like that, so there is two indexes on the same elasticsearch by default (will prevent a lot of mistakes imho).